torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1741,6 +1751,17 @@ def floor_div_handler(*args):
             expr = expr.replace(FloorDiv, floor_div_handler)
         return expr
 
+    def _enumerate_sympy_functions(self):
+        module = torch.utils._sympy.functions
+        all_functions = {getattr(module, func) for func in module.__all__}
+        self._unsupported_sympy_functions = all_functions.difference(self._supported_sympy_functions)
+
+    def _has_unsupported_sympy_function(self, expr) -> bool:
```

Comment:
@avikchaudhuri sorry, adding this back. I know you had this in the first place, I just didn't fully understand it at the time.